po/R-zh_CN.po

```diff
@@ -416,18 +478,33 @@ msgid ""
 msgstr ""
 "但i是一个 data.table (或者是字符向量)，必须使用 'on=' 参数指明参与连接的列 "
 "(参见 ?data.table)，可以是keying x(比如，已排序过，和标记已排序过，请参见?"
-"setkey)，或者是在x和i共用列的名字(比如，自然连接)。如果x有在内存被排序过，"
-"Keyed连接的速度会在非常大的数据上有较明显的提高。"
+"setkey)，或者是在x和i共用列的名字(比如，自然连接)。如果x有在内存被排序过，键"
+"(keyed)连接的速度会在非常大的数据上有较明显的提高。"
 
 msgid "Attempting to do natural join but no common columns in provided tables"
 msgstr "尝试进行自然连接然而并没有找到表格中相同的列"
 
-msgid "Internal error. Cannot by=.EACHI when joining to a secondary key, yet"
-msgstr "内部错误：目前尚无法对次键使用by=.EACH命令"
+msgid "Joining but 'x' has no key, natural join using"
+msgstr "联结但 'x' 没有键 (key)，自然联结用"
+
+msgid "not-join called with 'by=.EACHI'; Replacing !i with i=setdiff_(x,i) ..."
+msgstr ""
+
+msgid "Constructing irows for '!byjoin || nqbyjoin' ..."
+msgstr "造成 irows 与 '!byjoin || nqbyjoin' ..."
+
+msgid "Internal error. Cannot by=.EACHI when joining to an index, yet"
+msgstr "内部错误：目前尚无法对索引(index)使用by=.EACH命令"
 
 msgid "Internal error. irows has length in by=.EACHI"
 msgstr "内部错误：by=.EACHI 中 irows 有长度"
 
+msgid "Reorder irows for 'mult==\"all\" && !allGrp1' ..."
+msgstr "对'mult==\"all\" && !allGrp1'再排序irows ..."
+
+msgid "Reordering %d rows after bmerge done in ..."
+msgstr "再排序%d行。bmerge用了..."
```

Comment:
msgstr "bmerge用了... 之后再排序%d行。"

Reply:
Hmm a bit tough here, this will appear in logs like

```
Reordering 1000 rows after bmerge done in ... 3s
```

The message will show up before starting, so the `...` is like "pending" and then the timing shows up afterwards.

Maybe `bmerge之后再排序%d行用了...`?

Reply:
Yep, that sounds just right.